handler.go

```diff
@@ -1603,3 +1604,65 @@ func (h *Handler) handleDeleteInputDefinition(w http.ResponseWriter, r *http.Req
 }
 
 type postInputDefinitionResponse struct{}
+
+// MapAction Process the input data and set a bit
+func (h *Handler) MapAction(a *Action, value string, colID uint64) (*Bit, error) {
+	var bit Bit
```

Comment:
Since all actions take the same args and return *Bit, should we have 1 function handleAction which take action's name arg and handle each action based on different action's name, something like:
``` 
func (h *Handler) HandleAction(a *Action, value string, colID uint64) (*Bit, error){
             switch a.ValueDestination {
                       case Mapping:
                       case ValueToRow:
              }
}
```